### PR TITLE
Add more badges to readme

### PR DESCRIPTION
**Pull request by dstansby**
This adds badges required by PyOpenSci (xref https://github.com/sunpy/sunpy/issues/7060)

### CODE REVIEW

**Comment by Cadair** [comment]
I propose we drop the binder badge.

File: README.rst [hunk 1]
```diff
@@ -2,20 +2,33 @@
...
 .. |Binder| image:: https://mybinder.org/badge_logo.svg
    :target: https://mybinder.org/v2/gh/sunpy/sunpy/main?filepath=examples
 .. |Powered by NumFOCUS| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
    :target: https://numfocus.org
+.. |CI| image:: https://github.com/sunpy/sunpy/actions/workflows/ci.yml/badge.svg?branch=main
+   :target: https://github.com/sunpy/sunpy/actions/workflows/ci.yml
+.. |Docs| image:: https://readthedocs.org/projects/sunpy/badge/?version=stable
+   :target: https://docs.sunpy.org/en/stable/?badge=stable
+   :alt: Documentation Status
+.. |repostatus| image:: https://www.repostatus.org/badges/latest/active.svg
```
Comment (Cadair) [inline README.rst:27]: This feels somewhat pointless tbh. :shrug:

File: README.rst [hunk 2]
```diff
@@ -2,20 +2,33 @@
 ``sunpy``
 *********
 
-|Latest Version| |DOI| |matrix| |codecov| |Binder| |Powered by NumFOCUS|
+|Latest Version| |DOI| |repostatus| |python|
+|ci| |codecov| |Docs|
+|matrix|  |Binder|
+|Powered by NumFOCUS|
```
Comment (Cadair) [inline README.rst:8]: If you wanted to make these appear on different lines you didn't succeed according to the preview.

**Review by Cadair — APPROVED** [review]

File: README.rst (hunk 2 above)
Comment (dstansby) [reply README.rst:8]: I didn't, just thought in the source it was a bit clearer to group them.

**Pull request merged**